README.md

```diff
@@ -524,8 +524,8 @@ client.images
 * `client.images.get(image.id, [callback])`
 * `client.images.delete(image.id, [callback])`
 * `client.images.update(image.id, attributes, [callback])`
-* `client.images.listActions([page, perPage], image.id, [callback])`
-* `client.images.listActions([queryObject], image.id, [callback])`
+* `client.images.listActions(image.id, [page, perPage], [callback])`
```

Comment:
If I understand correctly, the image ID parameter should come before the pagination object specifier.